build.gradle

```diff
@@ -45,15 +45,31 @@ if (version.endsWith('-SNAPSHOT') && ext.buildNumber != null) {
   version = version.replace('-SNAPSHOT', versionSuffix)
 }
 
+ext {
+  release = project.hasProperty('release') && project.getProperty('release')
+  official = project.hasProperty('official') && project.getProperty('official')
+}
+
 repositories {
   jcenter()
   mavenLocal()
   mavenCentral()
   maven {
     url "https://maven.google.com"
   }
+  def repository = project.hasProperty('qa') ? 'sonarsource-qa' : 'sonarsource'
   maven {
-    url "https://repox.sonarsource.com/sonarsource"
+    url "https://repox.jfrog.io/repox/${repository}"
```

Comment:
jcenter and mavenCentral should be removed because they are mirrored by repox.
And are you sure that `mavenLocal` should be kept? It's [not declared in SonarQube](https://github.com/SonarSource/sonarqube/blob/master/build.gradle#L75)

Reply:
I'll try to remove and see if it works

Reply:
@simonbrandhof All right looks like it is working fine without those.